library/include/rocwmma/internal/blend.hpp

```diff
@@ -32,6 +32,9 @@
 #include "vector.hpp"
 // clang-format on
 
+using std::decay_t;
```

Comment:
This will need to back out and use the rocwmma::decay_t

Reply:
yeah, sure. will remove this line after rebasing on develop.